scripts/automated-patch-releases.md

```diff
@@ -0,0 +1,16 @@
+# CRI-O Automated Patch Releases
```

Comment:
Please include the mermaid diagrams directly here: https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/

Reply:
this link is not resolving `https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/`

Or do you mean I should copy the url of the automated-patch-releases.mermaid file and add it into this markdown file?

Reply:
The link outlines only how to include mermaid in GitHub markdown. We do not need the `.mermaid` files, we can just include everything in one markdown file.

Reply:
I think it looks good now